api/src/main/java/io/opencensus/resource/Resource.java

```diff
@@ -116,8 +141,8 @@ private static Resource createInternal(@Nullable String type, Map<String, String
    * <p>OC_RESOURCE_TYPE: A string that describes the type of the resource prefixed by a domain
    * namespace, e.g. “kubernetes.io/container”.
    */
-  @Nullable
-  static String parseResourceType(@Nullable String rawEnvType) {
+  /*@Nullable*/
```

Comment:
I am lost. When do we use the annotation and when do we use the comment?

Reply:
Yes, for public APIs we should use `@javax.annotations.Nullable`. For variables and internal methods, the comment one `/*@Nullable*/` is fine.

Reply:
What is the difference? Why are we not consistent everywhere?

Reply:
In the code most of the time they are the same except that `/*@Nullable*/` can be applied to generics. When building with Gradle, `@javax.annotations.Nullable` will be used by both error-prone and nullness checker, while `/*@Nullable*/` can only be used by nullness checker.

Generally speaking we can use `@javax.annotations.Nullable` whenever possible and only use `/*@Nullable*/` for generics. IMO this is beyond the scope of this PR though.

Reply:
So here it should be the annotation not the comment.